### ISSUE TITLE
GUI not accessible on Linux, exits silently

### ISSUE THREAD

**Issue by MagicD3VIL** [pixeltris/GK6X#192]
Greetings,
I am running the application via `sudo mono ./GK6X.exe /gui` on my Linux machine. The application starts correctly with an output of
```bash
Connected to device 'SK96S RGB' model:656801880 fw:v1.15
Started web GUI server at http://localhost:6464
```
but when I try to access the URL in my Firefox browser, it seems that the server is not running at all.

Moreover, the application prints out after about a half minute `Disconnected from device 'SK96S RGB'` and exits. There are no errors or other output to be found in the log file.

My Mono version is `Mono JIT compiler version 6.10.0.104 (tarball Wed Sep 23 10:35:39 UTC 2020)`.

Cheers

**Comment by MagicD3VIL** [comment]
It seems that the software didn't like I was connected to OpenVPN.

After checking with `netstat -tulpn`, I noticed that the `mono` was running on `10.8.0.2` instead of `127.0.0.1`. However, when I tried entering `http://10.8.0.2:6464` into my browser, I got an error saying `Bad Request (Invalid host)`.

I had to appropriately edit the `/etc/hosts` file. Closing as this was a user-config-related issue.
